static/themes/default.css

```diff
@@ -146,7 +146,7 @@ kbd {
 
 @media not print {
   #ag-editor-id {
-    max-width: var(--editorAreaWidth);
+    max-width: 100%;
```

Comment:
Is there a reason for this change? it seems unrelated to this PR, and causes the editor to stretch out to the entire screen on ultra wide monitors.